tests/lib/rules/prefer-stateless-function.js

```diff
@@ -345,6 +345,20 @@ ruleTester.run('prefer-stateless-function', rule, {
       errors: [{
         message: 'Component should be written as a pure function'
       }]
+    }, {
+      code: `
+        class Foo extends React.PureComponent {
+          render() {
+            return <div>this.bar</div>;
```

Comment:
Is this line missing curlies around `this.bar`?

Reply:
good catch, yes.